kotlin/.idea/misc.xml

```diff
@@ -13,6 +13,7 @@
       <item index="3" class="java.lang.String" itemvalue="org.openjdk.jmh.annotations.BenchmarkMode" />
     </list>
   </component>
+  <component name="ExternalStorageConfigurationManager" enabled="true" />
```

Comment:
Accidental, but I guess inevitable? This okay?

Reply:
This is the second PR to add this. What intellij version are you using? 

But I guess it's fine.

Reply:
Oh, derp, I'm using AS 4.0 Canary 6. I'll delete out of an abundance of caution.